_posts/2015-01-20-how-to-use-agrc-base-maps-in-qgis.md

```diff
@@ -5,20 +5,19 @@ author:
 tags:
   - base maps
   - open source
-date: 2015-01-20 07:51:04 -0700
+date: 2019-08-16 07:51:04 -0700
```

Comment:
There's an updated front matter tag. Let's use that and keep the original date the same.